run_serverless.sh

```
@@ -7,22 +7,37 @@ BOT_UUID=$(uuidgen)
+    "local_recording_server_location": "docker",
+    "automatic_leave": {
+        "waiting_room_timeout": 600,
+        "noone_joined_timeout": 600
+    },
+    "mp4_s3_path": "recordings/output.mp4",
+    "custom_branding_bot_path": "https://i.ibb.co/N9YtnDZ/ducobu.jpg",
+    "environ": "local",
+    "aws_s3_temporary_audio_bucket": "local-audio-bucket",
+    "remote": null,
+    "secret": "your-secret-key"
 }'
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)